Papyrus/Sources/Coders.swift

```diff
@@ -2,11 +2,11 @@ enum Coders {
 
     // MARK: HTTP Body
 
-    static var defaultHTTPBodyEncoder: HTTPBodyEncoder = .json()
-    static var defaultHTTPBodyDecoder: HTTPBodyDecoder = .json()
+    static let defaultHTTPBodyEncoder: HTTPBodyEncoder = .json()
+    static let defaultHTTPBodyDecoder: HTTPBodyDecoder = .json()
```

Comment:
Hmm - I wanted these to be modifiable so user could change default encoding with `Coders.defaultHTTPBodyEncoder = ...` when setting up their program. Unless seriously abused this wouldn't have a race condition - is there a way to just ignore the warning for this?

Reply:
Ah, because they were currently internal, I assumed it was just to avoid having it on multiple places but not necessary for users to be able to change these coders.

Reply:
Ha whoops - yep good point those weren't public in the first place.

Let's use your update and I can update to public access in the future.